templates/gitlab/README.md

```diff
@@ -1,6 +1,6 @@
 # GitLab CI Templates for Mender
 
-`mendersoftware/mender-ci-tools` image is used for the jobs. It's based on `alpine` with pre-installed `mender-cli` and `mender-artifact` tools.
+`mendersoftware/mender-ci-tools` image with pre-installed `mender-cli` and `mender-artifact` tools is used for the templates.
```

Comment:
Thank you :+1:

Reply:
It's actually is not used for the templates. It's used for the examples, but not for the templates. Templates here are [stage templates](https://learn.microsoft.com/en-us/azure/devops/pipelines/process/templates?view=azure-devops#stage-reuse), they depends only from `bash` and `curl` and are independent from the image.

Reply:
This is on GitLab README and AFAIK there we use the image.

In any case my thumbs up was for removing the `alpine` mention, which is something I had locally and I was about to submit when I saw you last push on this PR :)